cmd/kubeadm/app/apis/kubeadm/v1beta1/doc.go

```diff
@@ -212,6 +212,7 @@ limitations under the License.
 //	  certSANs:
 //	  - "10.100.1.1"
 //	  - "ec2-10-100-0-1.compute-1.amazonaws.com"
+//	  timeoutForControlPlane: 3m0s
```

Comment:
should we use the default here 4m?

Reply:
I don't think it's necessary.

Reply:
I agree with @neolit123, please change to 4 b/c that's the default used all over.